DotnetCLIVersion.txt

```diff
@@ -1 +1 @@
-2.2.300
+3.0.100-preview7-012821
```

Comment:
Remove this file? Should be able to use `global.json` now I think.

Reply:
Removed.